### PR TITLE
Adding Catppuccin Flavors (Themes)

### PR DESCRIPTION
**Pull request by rogeruiz**
Hi. I'd like to get these four themes into the official themes archive for _The Archive_. To learn more about Catppuccin, [view their website here](https://catppuccin.com).

## Previews

<details open>
<summary>🌻 Latte</summary>
latte

</details>
<details open>
<summary>🪴 Frappé</summary>
frappe

</details>
<details open>
<summary>🌺 Macchiato</summary>
macchiato

</details>
<details open>
<summary>🌿 Mocha</summary>
mocha

</details>

### CODE REVIEW

**Comment by rogeruiz** [comment]
Hey @DivineDominion is there any chance this could be reviewed and merged in soon? I've gotten approval from the Catppuccin organization. I can include screenshots as well. But they're accessible at catppuccin/catppuccin#2519 in the description and the repository called `the-archive` in the Catppuccin organization.

Thank you!

File: Themes/catppuccin-frappe.theme.json [hunk 1]
```diff
@@ -0,0 +1,171 @@
+{
+  "name": "Catppuccin Frappé",
+  "style": "Dark",
+  "author": [
+    {
+      "name": "Roger Steve Ruiz",
+      "url": "https://rog.gr/",
+      "basedOn": "https://github.com/catppuccin/catppuccin"
```
Comment (sgoudham) [inline Themes/catppuccin-frappe.theme.json:8]: I think it'd be nice to link to the repository itself.

```suggestion
      "basedOn": "https://github.com/catppuccin/the-archive"
```

File: Themes/catppuccin-frappe.theme.json [hunk 2]
```diff
@@ -0,0 +1,171 @@
+{
+  "name": "Catppuccin Frappé",
+  "style": "Dark",
+  "author": [
+    {
+      "name": "Roger Steve Ruiz",
+      "url": "https://rog.gr/",
+      "basedOn": "https://github.com/catppuccin/the-archive"
+    }
+  ],
```
Comment (DivineDominion) [inline Themes/catppuccin-frappe.theme.json:10]: Maybe add another JSON object here to link to the original Catppuccin repo or mention the original author(s) instead? As far as I'm concerned, your name on top as the author of this file, then something to acknowledge the community behind it?

```suggestion
  "author": [
    {
      "name": "Roger Steve Ruiz",
      "url": "https://rog.gr/"
    },
    {
      "name": "Catppuccin Organization",
      "url": "https://github.com/catppuccin/the-archive"
    }
  ],
```

Something like that would be parseable and visible in the app; the "basedOn" key would not be a.t.m. and just be flavor for readers of this code :)

File: Themes/catppuccin-latte.theme.json [hunk 3]
```diff
@@ -0,0 +1,171 @@
+{
+  "name": "Catppuccin Latte",
+  "style": "Light",
+  "author": [
+    {
+      "name": "Roger Steve Ruiz",
+      "url": "https://rog.gr/",
+      "basedOn": "https://github.com/catppuccin/catppuccin"
+    }
+  ],
```
Comment (DivineDominion) [inline Themes/catppuccin-latte.theme.json:10]: (Same here; slight inconsistency with URLs right now)

**Review by DivineDominion — CHANGES_REQUESTED** [review]
Thank you! Looks great.

Could you copy the `catppuccin/the-archive` repo's screenshot to your original PR comment at the top? We're not currently displaying screenshots in the app, but want to :)

File: Themes/catppuccin-macchiato.theme.json [hunk 4]
```diff
@@ -0,0 +1,171 @@
+{
+  "name": "Catppuccin Macchiato",
+  "style": "Dark",
+  "author": [
+    {
+      "name": "Roger Steve Ruiz",
+      "url": "https://rog.gr/",
+      "basedOn": "https://github.com/catppuccin/catppuccin"
+    }
+  ],
```
Comment (DivineDominion) [inline Themes/catppuccin-macchiato.theme.json:10]: (Same as above, author/URL inconsistency)

File: Themes/catppuccin-mocha.theme.json [hunk 5]
```diff
@@ -0,0 +1,171 @@
+{
+  "name": "Catppuccin Mocha",
+  "style": "Dark",
+  "author": [
+    {
+      "name": "Roger Steve Ruiz",
+      "url": "https://rog.gr/",
+      "basedOn": "https://github.com/catppuccin/catppuccin"
+    }
+  ],
```
Comment (DivineDominion) [inline Themes/catppuccin-mocha.theme.json:10]: (Same as above, author/URL inconsistency)

**Comment by julve** [comment]
No problem incorporating this instead of mine @DivineDominion. Thanks @rogeruiz!

**Review by DivineDominion — APPROVED** [review]
Thank you, that's great!

**Pull request merged**